conf/messages

```diff
@@ -1012,17 +1012,19 @@ estimatedIncome.error.tooSmall=Estimated income must be a number between 0 and 5
 estimatedIncome.error.tooLarge=Estimated income must be a number between 0 and 50,000,000,000.00
 estimatedIncome.error.tooManyDecimals=Estimated income must be 2 decimal places or less, for example 23,040.50 or 76,000
 
-personalAllowance.title=How much of your Personal Allowance do you want to apply?
-personalAllowance.helpText=Based on your income for the tax year {0} to {1}, you can apply up to {2}.
+personalAllowance.title=What is your Personal Allowance for tax year {0} to {1}?
+personalAllowance.helpText=	We''ll use this amount to work out the rate of Capital Gains Tax you should pay. The UK Personal Allowance in tax year {0} to {1} is {2}. Your Personal Allowance may be more than this if you receive Marriage Allowance or Blind Person’s Allowance. It will be less if your income is over {3}.
 personalAllowance.link=<a href="{0}" target="_blank">Find out about Personal Allowances (opens in a new window)</a>.
 personalAllowance.error.required=Enter how much of your Personal Allowance you want to apply
 personalAllowance.error.invalid=Personal Allowance amount must only include numbers, for example 12,500 or 207.50
 personalAllowance.error.tooSmall=Personal Allowance amount must be a number between 0 and {0}
-personalAllowance.error.tooLarge=Personal Allowance amount must be a number between 0 and {0}
+personalAllowance.error.tooLarge=User can only enter a maximum of £20,000 in this field for tax year 2020/2021
```

Comment:
the amount of money should be a parameter, as should the years

Reply:
Done, this is the one that took my time as there were two places, one in code and one in validator, and that was why the changes weren't taking effect. I changed it to a single place to generate the errors.